invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/ItemDetails/components/ItemMetadata/ItemMetadata.js

```diff
@@ -32,6 +46,16 @@ export default class ItemMetadata extends Component {
           <EditButton
             clickHandler={() => openRecordEditor(itemApi.url, item.item_pid)}
           />
+          <DeleteRecordModal
+            headerContent={`Are you sure you want to delete the Item
+            record with ID ${item.item_pid}?`}
+            onDelete={() => this.deleteItem(item.item_pid)}
+            refType={'Loan'}
+            onRefClick={this._handleOnRefClick}
+            checkRefs={() =>
+              loanApi.list(`item_pid:${item.item_pid} AND state:ITEM_ON_LOAN`)
```

Comment:
`state:ITEM_ON_LOAN`, I think we should use the config for `loanActiveStates` and maybe `PENDING` too? This is to avoid to delete an item when we have a `PENDING` loan on it.

Reply:
Is this still valid? @ntarocco @topless

Reply:
@zzacharo I think the safest way to treat this is to include pending loans also in the list of loans that prevent you to delete stuff.

Reply:
No sorry I meant if is still valid to filter by `loanActiveStates` and not just `state:ITEM_ON_LOAN `